src/structures/area.jl

```diff
@@ -58,12 +56,11 @@ A `GeoAvailability` is required if transmission should be included between indiv
 `Area`s due to a changed mass balance.
 
 # Fields
-- **`id`** is the name/identifier of the node.\n
-- **`input::Array{<:Resource}`** are the input `Resource`s with conversion value `Real`. \
-The latter are not relevant but included for consistency with other formulations.\n
-- **`output::Array{<:Resource}`** are the generated `Resource`s with conversion value `Real`. \
-The latter are not relevant but included for consistency with other formulations.\n
-
+- **`id`** is the name/identifier of the node.
+- **`input::Array{<:Resource}`** are the input `Resource`s with conversion value `Real`.
+  The latter are not relevant but included for consistency with other formulations.
```

Comment:
Is this still true when its an array? Or something that is left from when it was a dict?

Reply:
True. That has to be fixed by. Will do.